### PR TITLE
updated install instructions to new release 700

### PR DESCRIPTION
**Pull request by santhoshb-msft**
updated install instructions to new release 700

### CODE REVIEW

**Review by msalemcode — APPROVED** [review]

File: docs/Installation-Instructions.md
Comment (code4clouds) [inline docs/Installation-Instructions.md:1]: We need the upgrade instructions...

Comment (code4clouds) [reply docs/Installation-Instructions.md:1]: Add a new code segment to facilitate (copy/paste) of the previous version to the new one... in the spirit of "do not make me think".

Comment (santhoshb-msft) [reply docs/Installation-Instructions.md:1]: sure happy to do this as an enhancement. :-)

Comment (santhoshb-msft) [reply docs/Installation-Instructions.md:1]: Added issue. 471

**Pull request merged**